### PR TITLE
Add sprites with offline indication when user is not connected

### PR DESCRIPTION
**Pull request by JasonNoonan**
Added some simple sprites, and modified the quest display to make better use of them.

### CODE REVIEW

File: lib/point_quest_web/live/quest.ex [hunk 1]
```diff
@@ -22,15 +22,19 @@ defmodule PointQuestWeb.QuestLive do
...
-
-      <div>
-        <pre><code><%= Jason.encode!(Ecto.embedded_dump(@quest, :json), pretty: true) %></code></pre>
-      </div>
       <div class="flex gap-4">
-        <div :for={{user_id, %{name: name, class: class}} <- @users} class="bg-blue-400">
-          <%= user_id %>
-          <%= name %>
-          <%= class %>
+        <!-- <div :for={{user_id, %{name: name, class: class}} <- @users} class="flex flex-col p-2"> -->
```
Comment (skbolton) [inline lib/point_quest_web/live/quest.ex]: ⚒️

File: lib/point_quest_web/live/quest.ex [hunk 2]
```diff
@@ -22,15 +22,19 @@ defmodule PointQuestWeb.QuestLive do
...
+        <div :for={%{id: id, class: class, name: name} <- @adventurers} class="flex flex-col p-2">
+          <div class={"#{get_background_color(id, @users)} rounded p-2"}>
+            <div id="class-sprite" class="w-[64px] h-[64px] mt-2">
```
Comment (skbolton) [inline lib/point_quest_web/live/quest.ex]: `w-[64px]` is the same as `w-16` and we can save a custom tailwind class.

File: lib/point_quest_web/live/quest.ex [hunk 3]
```diff
@@ -147,4 +163,18 @@ defmodule PointQuestWeb.QuestLive do
...
   defp show_attack_panel?(%PartyLeader{adventurer: nil}), do: false
   defp show_attack_panel?(_actor), do: true
+
+  defp get_background_color(adventurer_id, users) do
+    case Map.get(users, adventurer_id) do
+      nil ->
+        "bg-gray-400"
+
+      user ->
+        if user.connected? do
```
Comment (skbolton) [inline lib/point_quest_web/live/quest.ex]: You can save a nested if here.

File: priv/static/images/mage.png
Comment (skbolton) [inline priv/static/images/mage.png:1]: well fuck me, I am a mage.

File: lib/point_quest_web/live/quest.ex (hunk 3 above)
Comment (skbolton) [reply lib/point_quest_web/live/quest.ex]: Oh wait looked at it wrong. You would need a weird `cond` to do it.

File: lib/point_quest_web/live/quest.ex (hunk 1 above)
Comment (JasonNoonan) [reply lib/point_quest_web/live/quest.ex]: Whoops, thanks. Getting sloppy in my old age...

File: lib/point_quest_web/live/quest.ex (hunk 2 above)
Comment (JasonNoonan) [reply lib/point_quest_web/live/quest.ex]: nice! will update shortly

File: priv/static/images/mage.png
Comment (JasonNoonan) [reply priv/static/images/mage.png:1]: time to go bald apparently

**Review by skbolton — APPROVED** [review]

**Pull request merged**